optimizer/lib/transformers/AddBlurryImagePlaceholders.js

```diff
@@ -74,15 +74,27 @@ class AddBlurryImagePlaceholders {
    * @param {TreeAdapter} tree A parse5 treeAdapter.
    * @param {String} src The image that the bitmap is based on.
    * @return {!Promise} A promise that signifies that the img has been updated
-   * to have correct attributes to be a blurred placeholder.
+   * to have correct attributes to be a blurred placeholder along with the
+   * placeholder itself.
    * @private
    */
   addBlurryPlaceholder_(tree, src) {
     const img = tree.createElement('img');
-    img.attribs.src = src;
-    img.attribs.class = 'i-amphtml-blur';
+    img.attribs.class = 'i-amphtml-blurry-placeholder';
     img.attribs.placeholder = '';
-    return this.getDataURI_(img).then(() => {
+    img.attribs.src = src;
+    return this.getDataURI_(img).then((dataURI) => {
+      let svg = '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://ww'
+      + `w.w3.org/1999/xlink" viewBox="0 0 ${dataURI.width} ${dataURI.height}">`
+      + '<filter id="b" color-interpolation-filters="sRGB"><feGaussianBlur stdD'
+      + 'eviation=".5"></feGaussianBlur><feComponentTransfer><feFuncA type="dis'
+      + 'crete" tableValues="1 1%0A"></feFuncA></feComponentTransfer></filter><'
+      + 'image filter="url(#b)" x="0" y="0" height="100%" width="100%" xlink:hr'
+      + `ef="${dataURI.src}"></image></svg>`;
```

Comment:
Can we avoid line breaks in the middle of an attribute and add line breaks after children?

```js
const svgOpen = '<svg xmlns="http://www.w3.org/2000/svg" ' 
    + 'xmlns:xlink="http://www.w3.org/1999/xlink"' 
    + `viewBox="0 0 ${dataURI.width} ${dataURI.height}">`;
const filter = '<filter id="b" color-interpolation-filters="sRGB">';
const blur = '<feGaussianBlur stdDeviation=".5"></feGaussianBlur>';

const html = svgOpen + filter + blur + ...;
```